.gitignore

```diff
@@ -83,6 +83,9 @@ nbactions*.xml
 /gen4/
 /tool/playground/
 tmp/
+**/generatedCode/*.interp
```

Comment:
I guess **/generatedCode/* would be better ?

Reply:
A single leading asterisk will not match the slash character. The double asterisk will and is therefore needed to match `generatedCode` in nested directories.

The reason for listing three different file extensions is that I don't want to exclude the entire `generatedCode` directory, as it contains .js files for the toy grammars and we need those for the tests.